test/helpers/default-functions.ts

```diff
@@ -140,7 +140,7 @@ export function defaultFunctions(): void {
 			await browser.waitUntil(
 				async () => {
 					const commandTextArray = await Promise.all(
-						commands.map( async ( command ) => command.getText() )
+						await commands.map( async ( command ) => command.getText() )
```

Comment:
This is why bitbuttler DOES have a product. Such things in another PR would be great. But also fine with me as long as it works...

Reply:
In this case this change is here to make TS transpilation not fail after udpated dependencies. It actually doesn't work as is and is a WIP...

Reply:
So looping back here... Given that the QuickStatements tests all pass it does seem that this thing that Typescript complained about and I changed accordingly was necessary. In the dependency updates either the `$$` return type changed in WDIO (which from looking through release notes I think they did), or the TS settings got more picky.  Either way it is doing the right thing now :)